lib/event.rb

```diff
@@ -0,0 +1,71 @@
+require 'pry'
+
+
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    @ages.max
+  end
+
+  def min_age
+    @ages.min
+  end
+
+  def sum_ages
+    sum = 0
+    @ages.each do |a|
+      sum = sum + a
+    end
+    return sum
+  end
+
+  def average_age
+    sum = sum_ages.to_f
+    average = (sum / (@ages.count)).round(2)
+    return average
+  end
+
+  def standard_deviation_age
+    mean = average_age
+    ages_mod1 = ages_minus_average(mean)
+    ages_mod2 = square_each_modified_age(ages_mod1)
+    summed_modified_ages = sum_squared_mod_ages(ages_mod2)
+    mean_modified_age = average_modified_ages(summed_modified_ages)
+    std = square_root(mean_modified_age)
```

Comment:
This totally works - you could refactor to `return square_root(mean_modified_age)` to slim the method by one line.